src/main/java/com/stripe/param/PriceListParams.java

```diff
@@ -0,0 +1,568 @@
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class PriceListParams extends ApiRequestParams {
+  /**
+   * Only return prices that are active or inactive (e.g., pass {@code false} to list all inactive
+   * prices).
+   */
+  @SerializedName("active")
+  Boolean active;
+
+  /**
+   * A filter on the list, based on the object {@code created} field. The value can be a string with
+   * an integer Unix timestamp, or it can be a dictionary with a number of different query options.
+   */
+  @SerializedName("created")
+  Object created;
+
+  /** Only return prices for the given currency. */
+  @SerializedName("currency")
+  String currency;
+
+  /**
+   * A cursor for use in pagination. {@code ending_before} is an object ID that defines your place
+   * in the list. For instance, if you make a list request and receive 100 objects, starting with
+   * {@code obj_bar}, your subsequent call can include {@code ending_before=obj_bar} in order to
+   * fetch the previous page of the list.
+   */
+  @SerializedName("ending_before")
+  String endingBefore;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /**
+   * Map of extra parameters for custom features not available in this client library. The content
+   * in this map is not serialized under this field's {@code @SerializedName} value. Instead, each
+   * key/value pair is serialized as if the key is a root-level field (serialized) name in this
+   * param object. Effectively, this map is flattened to its parent instance.
+   */
+  @SerializedName(ApiRequestParams.EXTRA_PARAMS_KEY)
+  Map<String, Object> extraParams;
+
+  /**
+   * A limit on the number of objects to be returned. Limit can range between 1 and 100, and the
+   * default is 10.
+   */
+  @SerializedName("limit")
+  Long limit;
+
+  /** Only return the price with these lookup_keys, if any exist. */
+  @SerializedName("lookup_key")
+  List<String> lookupKey;
```

Comment:
Is this expected to be a list of strings? And if so should it be called `lookupKeys` instead?